pkg/volume/util/types/types.go

```diff
@@ -121,3 +122,81 @@ const (
 	// as a annotation to the PVC.
 	VolumeResizerKey = "volume.kubernetes.io/storage-resizer"
 )
+
+type OperationKey struct {
```

Comment:
I don't think this should be in the `util/types` package. Can we put it in the nestedPendingOperation package?

Reply:
Had it there originally, but the problem is OperationExecutor has `IsOperationPending()` that requires a key, and it's called in ADC reconciler.